docs/datasheets/dagw-wikibooks.md

```diff
@@ -0,0 +1,56 @@
+---
+pretty_name: Wikibooks
+language:
+  - da
+license: cc0-1.0
+license_name: Creative Commons Zero v1.0 Universal
+size_categories:
+  - 1-10k
+task_categories:
+  - text-generation
+  - fill-mask
+task_ids:
+  - language-modeling
+---
+# Dataset Card for Wikibooks
+## Dataset Description
+- **Number of records:** 1319
+- **Languages:** Danish
+## Dataset Sturcture
+An example from the dataset looks as follows.
+```yaml
+{
+    'text': 'Spilinfo.
+Spillet er lavet af Blizzard Entertainme',
```

Comment:
```suggestion
    'text': 'Spilinfo. Spillet er lavet af Blizzard Entertainme...
```
I would make is as so to make it clear that it is cut off

Reply:
yes, will do. So what I have done before is simply printing out the first 50 characters, I'll filter out '\n' first and add '...' in the end